### ISSUE TITLE
"Apply Config" does not restart wg0

### ISSUE THREAD

**Issue by matthewpipes** [ngoduykhanh/wireguard-ui#283]
As I understand it, "Apply Config" is supposed to stop and start the wg-quick@wg0 service. I've noticed that for my server it does not. If I add a new client and click "Apply Config" the new client does not show when I enter "sudo wg show wg0" in the cli. I must manually start and stop it. Any help or suggestions?

https://user-images.githubusercontent.com/64096239/209714283-3c419e69-48d1-4dde-a3a8-538f7a008018.mp4

**Comment by aghus** [comment]
I don't know if the UI has the funcionality written to do that, and if it does, you are correct that does not work.

For a workaround:

Run this command `/bin/wg syncconf wg0 <(/bin/wg-quick strip wg0)`

that line will run `wg-quick strip wg0` that will read the `wg0.conf` file and generate a configuration that `wg syncconf wg0` will understand and this will update the connections without needing to restart the service.

On issue #211 there is a way to detect the changes of the file `wg0.conf`, and automatically run the command to update the connections.

**Comment by systemcrash** [comment]
Did you follow the instructions in the readme to create the systemd scripts? This works fine for me on RockyLinux.

**Comment by matthewpipes** [comment]
> Did you follow the instructions in the readme to create the systemd scripts? This works fine for me on RockyLinux.

Yeah I did. I've resolved this issue a while ago but forgot to update here. I had to change `ExecStart=/usr/bin/systemctl restart wg-quick@wg0.service` to `ExecStart=systemctl restart wg-quick@wg0.service` in the `/etc/systemd/system/wgui.service` file for it to work correctly. It would error with `no such file or directory found`. Fixing that and doing a `systemctl daemon-reload` made it work correctly
